sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -291,6 +291,7 @@ class RocksDB(
     acquire(LoadStore)
     recordedMetrics = None
     logInfo(log"Loading ${MDC(LogKeys.VERSION_NUM, version)}")
+    rocksDBFileMapping.currentVersion = version
```

Comment:
nit: should we add a comment for why this is needed ?

Reply:
done